### ISSUE TITLE
DynamicEarthNet Labels

### ISSUE THREAD

**Issue by xuway** [ElliotVincent/SitsSCD#5]
Hi Dr. Vincent,

I see in the paper, your team reconstructed the DynamicEarthNet dataset in a memory-efficient format, which is greatly appreciated. I noticed that your remake dataset contains 75 labeled ROIs, whereas the official DynamicEarthNet website only provides semantic labels for 55 ROIs corresponding to the training set.

Could you clarify where the additional 20 labeled ROIs come from (official val & testset labels)? Are these ground-truth semantic maps provided by the dataset creators, or are they labels generated by other machine learning models or labeled by humans?

Thank you very much for your clarity.

**Comment by ElliotVincent** [comment]
Hi xuway,

The 20 validation/test labeled ROIs were provided by the dataset creators and are available [here](https://mediatum.ub.tum.de/1738088).
